### DISCUSSION TITLE
use inlineContainer with CldUploadWidget

### DISCUSSION THREAD

**Discussion by gabrielmopereira** [cloudinary-community/next-cloudinary#560] [Q&A]
Is it possible to use the component always rendered inside the page or do I have to manually create the widget (i.e not using this library components)?

**Comment by colbyfayock** [comment]
hey @gabriel-mopereira  can you try passing `inlineContainer` in as `options={{ inlineContainer: ... }}`?

**Reply by gabrielmopereira** [reply]
Hey @colbyfayock,

Thank you for replying! 
Yes, I've tried that before, but I didn't manage to have the widget rendered. I tried using useRef in a "brother" div and the document.getElementByID. Both failed.

**Reply by colbyfayock** [reply]
ah okay. im going to create an issue based off of this discussion. i unfortunately dont have a current timeline though for when it will get addressed

**Reply by gabrielmopereira** [reply]
That's ok. Thank you!
